test/testPlugin.js

```diff
@@ -1,9 +1,21 @@
-import { transform } from 'babel-core';
+import * as babel6 from 'babel-core';
+
+function babel7lazy() {
+  // This can't be imported in node 4.
+  // eslint-disable-next-line global-require
+  const babel7 = require('@babel/core');
+  // eslint-disable-next-line no-func-assign
+  babel7lazy = () => babel7;
```

Comment:
this doesn't seem particularly helpful, since there's a require cache - why not have this function be `() => require('@babel/core')`?

Reply:
Right. For some reason the `global-require` made me think that require in functions was bad because it was executed multiple times :sweat_smile:

Reply:
I believe it's meant mainly to stop non-unconditional, and discourage dynamic, requires :-)